addons/website_sale_stock/tests/test_website_sale_stock_product_template.py

```diff
@@ -0,0 +1,31 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+
+from odoo.tests import tagged, users
+
+from odoo.addons.website.tools import MockRequest
+from odoo.addons.website_sale_stock.tests.common import WebsiteSaleStockCommon
+
+
+@tagged('post_install', '-at_install')
+class TestWebsiteSaleStockProductTemplate(WebsiteSaleStockCommon):
+
+    @users('portal')
```

Comment:
Mhm, portal user is from the demo data and unreliable, should be `portal_user` and you should use `BaseUsersCommon` (or call the method `_create_new_portal_user` in the setupClass)

Reply:
Resolving as this is now outdated.